add blorp to list of apps

christianjuth opened the pull request:
I added MacOS as a platform. Figured it would be nice to let people filter by desktop platforms. But there isn't an icon for MacOS.

Nutomic commented:
As its available for web, mac and ios it would make more sense to put it under `All`, or potentially rename that option to `Multiplatform`.

christianjuth commented:
@Nutomic up to you. I reverted adding MacOS for now. I guess that doesn't really make sense anyway unless we added Linux and Windows as well.

The main issue is I decided to distribute my Mac app outside of the App Store, so I thought it might not be obvious that the Mac version is available. But I already mentioned it in the description and I have a download button on the website, so it's probably fine.

Also I hope my app description doesn't come across as disrespectful to anyone. I was trying to be funny since there seems to be more Lemmy clients than JavaScript frameworks at this point lol.

Nutomic commented:
The text sounds fine. For platforms another option would be to add a general variant `Desktop` instead of listing every single OS.

christianjuth commented:
@Nutomic I like the desktop option. Pretty sure Voyager will fall under that same category in the near future.

CC @aeharding

christianjuth commented:
@Nutomic how do you want to proceed? Can we merge this as is and highlight desktop apps later?

I'm happy to try it now, but I would need to add a new icon (idk where you pull icons from) and idk if it would be up to me to track down which apps support desktop. But I can try.

dessalines commented:
You don't need to add desktop to any other ones, just add the `Desktop` to platforms, and add it to yours. 

We can go through the others some other time.

christianjuth commented:
@Nutomic done!

Note:
* Icon was generate with ChatGPT just so you're aware incase that poses any issues
* Translations should be updated to include `"desktop"`

dessalines commented:
We can fix the icon later if its an issue. I'll add a translation now.

Nutomic commented:
Thanks!
